CRITICAL - 2025-08-15 22:19:21 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "welcome_message.php"
[Method: GET, Route: /]
in VENDORPATH\codeigniter4\framework\system\Exceptions\FrameworkException.php on line 39.
 1 VENDORPATH\codeigniter4\framework\system\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('welcome_message.php')
 2 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('welcome_message', [], true)
 3 APPPATH\Controllers\Home.php(9): view('welcome_message')
 4 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Home->index()
 5 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\rmb_web\\public\\index.php')
ERROR - 2025-08-15 22:20:24 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#4 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#5 C:\xampp\htdocs\rmb_web\app\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', Array)
#6 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
#7 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#8 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\rmb_web\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\rmb_web\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\rmb_web\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xampp\htdocs\rmb_web\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#3 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#4 C:\xampp\htdocs\rmb_web\app\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', Array)
#5 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
#6 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#7 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\rmb_web\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\rmb_web\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\rmb_web\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 C:\xampp\htdocs\rmb_web\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2025-08-15 22:20:24 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: /]
in VENDORPATH\codeigniter4\framework\system\Database\BaseConnection.php on line 465.
 1 VENDORPATH\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 VENDORPATH\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `products`', null, false)
 3 VENDORPATH\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 4 APPPATH\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', [])
 5 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
 6 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
 7 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\rmb_web\\public\\index.php')
ERROR - 2025-08-15 22:27:13 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#4 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#5 C:\xampp\htdocs\rmb_web\app\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', Array)
#6 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
#7 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#8 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\rmb_web\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\rmb_web\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\rmb_web\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xampp\htdocs\rmb_web\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#3 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#4 C:\xampp\htdocs\rmb_web\app\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', Array)
#5 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
#6 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#7 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\rmb_web\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\rmb_web\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\rmb_web\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 C:\xampp\htdocs\rmb_web\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2025-08-15 22:27:13 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: /]
in VENDORPATH\codeigniter4\framework\system\Database\BaseConnection.php on line 465.
 1 VENDORPATH\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 VENDORPATH\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `products`', null, false)
 3 VENDORPATH\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 4 APPPATH\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', [])
 5 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
 6 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
 7 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\rmb_web\\public\\index.php')
ERROR - 2025-08-15 22:36:18 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#4 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#5 C:\xampp\htdocs\rmb_web\app\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', Array)
#6 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
#7 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#8 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\rmb_web\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\rmb_web\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\rmb_web\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xampp\htdocs\rmb_web\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#3 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#4 C:\xampp\htdocs\rmb_web\app\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', Array)
#5 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
#6 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#7 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\rmb_web\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\rmb_web\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\rmb_web\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 C:\xampp\htdocs\rmb_web\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2025-08-15 22:36:18 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: /]
in VENDORPATH\codeigniter4\framework\system\Database\BaseConnection.php on line 465.
 1 VENDORPATH\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 VENDORPATH\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `products`', null, false)
 3 VENDORPATH\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 4 APPPATH\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', [])
 5 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
 6 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
 7 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\rmb_web\\public\\index.php')
ERROR - 2025-08-15 22:36:33 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#4 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#5 C:\xampp\htdocs\rmb_web\app\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', Array)
#6 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
#7 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#8 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\rmb_web\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\rmb_web\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\rmb_web\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xampp\htdocs\rmb_web\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#3 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#4 C:\xampp\htdocs\rmb_web\app\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', Array)
#5 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
#6 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#7 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\rmb_web\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\rmb_web\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\rmb_web\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 C:\xampp\htdocs\rmb_web\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2025-08-15 22:36:33 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: /]
in VENDORPATH\codeigniter4\framework\system\Database\BaseConnection.php on line 465.
 1 VENDORPATH\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 VENDORPATH\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `products`', null, false)
 3 VENDORPATH\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 4 APPPATH\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', [])
 5 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
 6 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
 7 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\rmb_web\\public\\index.php')
ERROR - 2025-08-15 22:36:45 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#4 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#5 C:\xampp\htdocs\rmb_web\app\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', Array)
#6 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
#7 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#8 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\rmb_web\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\rmb_web\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\rmb_web\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xampp\htdocs\rmb_web\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#3 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#4 C:\xampp\htdocs\rmb_web\app\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', Array)
#5 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
#6 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#7 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\rmb_web\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\rmb_web\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\rmb_web\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 C:\xampp\htdocs\rmb_web\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2025-08-15 22:36:45 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: /]
in VENDORPATH\codeigniter4\framework\system\Database\BaseConnection.php on line 465.
 1 VENDORPATH\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 VENDORPATH\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `products`', null, false)
 3 VENDORPATH\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 4 APPPATH\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', [])
 5 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
 6 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
 7 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\rmb_web\\public\\index.php')
ERROR - 2025-08-15 22:36:48 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#4 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#5 C:\xampp\htdocs\rmb_web\app\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', Array)
#6 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
#7 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#8 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\rmb_web\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\rmb_web\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\rmb_web\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xampp\htdocs\rmb_web\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#3 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#4 C:\xampp\htdocs\rmb_web\app\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', Array)
#5 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
#6 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#7 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\rmb_web\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\rmb_web\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\rmb_web\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 C:\xampp\htdocs\rmb_web\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2025-08-15 22:36:48 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: /]
in VENDORPATH\codeigniter4\framework\system\Database\BaseConnection.php on line 465.
 1 VENDORPATH\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 VENDORPATH\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `products`', null, false)
 3 VENDORPATH\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 4 APPPATH\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', [])
 5 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
 6 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
 7 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\rmb_web\\public\\index.php')
ERROR - 2025-08-15 22:36:55 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#4 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#5 C:\xampp\htdocs\rmb_web\app\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', Array)
#6 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
#7 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#8 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\rmb_web\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\rmb_web\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\rmb_web\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xampp\htdocs\rmb_web\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#3 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#4 C:\xampp\htdocs\rmb_web\app\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', Array)
#5 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
#6 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#7 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\rmb_web\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\rmb_web\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\rmb_web\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 C:\xampp\htdocs\rmb_web\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2025-08-15 22:36:55 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: /]
in VENDORPATH\codeigniter4\framework\system\Database\BaseConnection.php on line 465.
 1 VENDORPATH\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 VENDORPATH\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `products`', null, false)
 3 VENDORPATH\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 4 APPPATH\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', [])
 5 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
 6 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
 7 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\rmb_web\\public\\index.php')
ERROR - 2025-08-15 22:37:02 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#4 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#5 C:\xampp\htdocs\rmb_web\app\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', Array)
#6 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
#7 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#8 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\rmb_web\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\rmb_web\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\rmb_web\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xampp\htdocs\rmb_web\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#3 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#4 C:\xampp\htdocs\rmb_web\app\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', Array)
#5 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
#6 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#7 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\rmb_web\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\rmb_web\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\rmb_web\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 C:\xampp\htdocs\rmb_web\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2025-08-15 22:37:02 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: /]
in VENDORPATH\codeigniter4\framework\system\Database\BaseConnection.php on line 465.
 1 VENDORPATH\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 VENDORPATH\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `products`', null, false)
 3 VENDORPATH\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 4 APPPATH\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', [])
 5 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
 6 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
 7 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\rmb_web\\public\\index.php')
ERROR - 2025-08-15 22:37:44 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#4 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#5 C:\xampp\htdocs\rmb_web\app\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', Array)
#6 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
#7 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#8 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\rmb_web\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\rmb_web\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\rmb_web\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xampp\htdocs\rmb_web\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#3 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#4 C:\xampp\htdocs\rmb_web\app\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', Array)
#5 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
#6 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#7 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\rmb_web\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\rmb_web\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\rmb_web\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 C:\xampp\htdocs\rmb_web\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2025-08-15 22:37:44 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: /]
in VENDORPATH\codeigniter4\framework\system\Database\BaseConnection.php on line 465.
 1 VENDORPATH\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 VENDORPATH\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `products`', null, false)
 3 VENDORPATH\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 4 APPPATH\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', [])
 5 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
 6 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
 7 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\rmb_web\\public\\index.php')
ERROR - 2025-08-15 22:37:46 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#4 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#5 C:\xampp\htdocs\rmb_web\app\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', Array)
#6 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
#7 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#8 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\rmb_web\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\rmb_web\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\rmb_web\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xampp\htdocs\rmb_web\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#3 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#4 C:\xampp\htdocs\rmb_web\app\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', Array)
#5 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
#6 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#7 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\rmb_web\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\rmb_web\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\rmb_web\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 C:\xampp\htdocs\rmb_web\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2025-08-15 22:37:46 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: /]
in VENDORPATH\codeigniter4\framework\system\Database\BaseConnection.php on line 465.
 1 VENDORPATH\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 VENDORPATH\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `products`', null, false)
 3 VENDORPATH\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 4 APPPATH\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', [])
 5 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
 6 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
 7 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\rmb_web\\public\\index.php')
ERROR - 2025-08-15 22:38:20 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#4 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#5 C:\xampp\htdocs\rmb_web\app\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', Array)
#6 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
#7 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#8 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\rmb_web\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\rmb_web\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\rmb_web\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xampp\htdocs\rmb_web\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#3 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
#4 C:\xampp\htdocs\rmb_web\app\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', Array)
#5 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
#6 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#7 C:\xampp\htdocs\rmb_web\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\rmb_web\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\rmb_web\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\rmb_web\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 C:\xampp\htdocs\rmb_web\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2025-08-15 22:38:20 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: /]
in VENDORPATH\codeigniter4\framework\system\Database\BaseConnection.php on line 465.
 1 VENDORPATH\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 VENDORPATH\codeigniter4\framework\system\Database\BaseBuilder.php(1678): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `products`', null, false)
 3 VENDORPATH\codeigniter4\framework\system\Model.php(925): CodeIgniter\Database\BaseBuilder->countAll()
 4 APPPATH\Controllers\HomeController.php(29): CodeIgniter\Model->__call('countAll', [])
 5 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\HomeController->index()
 6 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
 7 VENDORPATH\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\rmb_web\\public\\index.php')
DEBUG - 2025-08-15 22:54:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-08-15 22:54:27 --> Error: Call to undefined function App\Filters\get_cookie()
[Method: GET, Route: admin]
in APPPATH\Filters\AdminAuthFilter.php on line 31.
 1 SYSTEMPATH\Filters\Filters.php(241): App\Filters\AdminAuthFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 2 SYSTEMPATH\Filters\Filters.php(221): CodeIgniter\Filters\Filters->runBefore([...])
 3 SYSTEMPATH\CodeIgniter.php(479): CodeIgniter\Filters\Filters->run('admin', 'before')
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\rmb_web\\public\\index.php')
DEBUG - 2025-08-15 22:56:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-08-15 22:56:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-08-15 22:58:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-08-15 22:58:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-08-15 22:58:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-08-15 22:58:30 --> Admin login successful: rainelson@gmail.com
DEBUG - 2025-08-15 22:58:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-08-15 22:58:31 --> Admin login successful: rainelson@gmail.com
DEBUG - 2025-08-15 22:58:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-08-15 22:58:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2025-08-15 22:58:34 --> [DEPRECATED] Creation of dynamic property App\Controllers\ProductsController::$productGalleryModel is deprecated in APPPATH\Controllers\ProductsController.php on line 19.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\ProductsController->__construct()
 2 SYSTEMPATH\CodeIgniter.php(498): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\rmb_web\\public\\index.php')
DEBUG - 2025-08-15 22:58:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2025-08-15 22:58:48 --> [DEPRECATED] Creation of dynamic property App\Controllers\ProductsController::$productGalleryModel is deprecated in APPPATH\Controllers\ProductsController.php on line 19.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\ProductsController->__construct()
 2 SYSTEMPATH\CodeIgniter.php(498): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\rmb_web\\public\\index.php')
DEBUG - 2025-08-15 22:58:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-08-15 23:08:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-08-15 23:08:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-08-15 23:08:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2025-08-15 23:08:59 --> [DEPRECATED] Creation of dynamic property App\Controllers\ProductsController::$productGalleryModel is deprecated in APPPATH\Controllers\ProductsController.php on line 19.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\ProductsController->__construct()
 2 SYSTEMPATH\CodeIgniter.php(498): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\rmb_web\\public\\index.php')
